### PR TITLE
Topic/dt feature search

### PR DESCRIPTION
**Pull request by nmenda**
rewrote feature search with dataTables
This replaces old feature search which does not work anymore, but eventually will keep only the gene/locus search tab and retire completely the feature table search

### CODE REVIEW

File: lib/SGN/Controller/Search/Feature.pm [hunk 1]
```diff
@@ -37,11 +46,13 @@ sub oldsearch : Path('/feature/search') Args(0) {
 
 sub search :Path('/search/features') Args(0) {
```
Comment (nickmorales) [inline lib/SGN/Controller/Search/Feature.pm]: There's two /search/features paths? (the others in SGN::Controller::Feature above). maybe one should be deprecated

File: mason/search/features.mas [hunk 2]
```diff
@@ -1,33 +1,138 @@
 <%args>
-  $maximum_export_size
+ $organisms         => undef
+ $feature_types     => undef
 </%args>
 
 <& /util/import_extjs.mas &>
```
Comment (nickmorales) [inline mason/search/features.mas]: this mason (import_extjs.mas) is trying to pull in js and css that no longer exists. maybe remove this line and deprecate that mason file?

File: lib/SGN/Controller/Search/Feature.pm (hunk 1 above)
Comment (nmenda) [inline lib/SGN/Controller/Search/Feature.pm]: looks like the older one in SGN::Controller::Feature needs to be deprecated

File: mason/search/features.mas (hunk 2 above)
Comment (nmenda) [inline mason/search/features.mas]: https://github.com/solgenomics/sgn/commit/e71aa45679806d6a7675f1c5985d9da175ef2af6

File: lib/SGN/Controller/Search/Feature.pm (hunk 1 above)
Comment (nmenda) [inline lib/SGN/Controller/Search/Feature.pm]: https://github.com/solgenomics/sgn/commit/3126858e6f2fc89a2371819bf7871e8d8a30f7a1

**Pull request merged**